storage_backend_s3/models/storage_backend.py

```diff
@@ -58,8 +61,17 @@ def _server_env_fields(self):
         return env_fields
 
     def _selection_aws_region(self):
+        # This calls AWS web servers,
+        # it introduce latency in rendering of
+        # form view and tree view of storage.backend
         session = boto3.session.Session()
         return [
             (region, region.replace("-", " ").capitalize())
             for region in session.get_available_regions("s3")
         ]
```

Comment:
I think it's better to store this in model cache instead.

```suggestion
    _aws_available_regions = None
    
    def _selection_aws_region(self):
        if self._aws_available_regions:
            return self._aws_available_regions
        # This calls AWS web servers,
        # it introduce latency in rendering of
        # form view and tree view of storage.backend
        session = boto3.session.Session()
        self._aws_available_regions = [
            (region, region.replace("-", " ").capitalize())
            for region in session.get_available_regions("s3")
        ]
        return self._aws_available_regions
```